=== tests/cases/conformance/types/never/neverTypeErrors1.ts ===
function f1() {
>f1 : () => 无值

    let x: never;
>x : 不及
>类型别名 :=> 不及

    x = 1;
>x = 1 : 1
>x : 不及
>类型别名 :=> 不及
>1 : 1

    x = "abc";
>x = "abc" : "abc"
>x : 不及
>类型别名 :=> 不及
>"abc" : "abc"

    x = false;
>x = false : 为假
>类型别名 :=> 为假
>x : 不及
>类型别名 :=> 不及
>false : 为假
>类型别名 :=> 为假

    x = undefined;
>x = undefined : 未定
>类型别名 :=> 未定
>x : 不及
>类型别名 :=> 不及
>undefined : 未定
>类型别名 :=> 未定

    x = null;
>x = null : 空值
>类型别名 :=> 空值
>x : 不及
>类型别名 :=> 不及
>null : 空值
>类型别名 :=> 空值

    x = {};
>x = {} : {}
>x : 不及
>类型别名 :=> 不及
>{} : {}
}

function f2(): never {
>f2 : () => 不及

    return;
}

function f3(): never {
>f3 : () => 不及

    return 1;
>1 : 1
}

function f4(): never {
>f4 : () => 不及
}
